### PR TITLE
command to check current user in lakectl

### LINKED ISSUE #8313: There is no lakectl command to check current user
This is an API that we lack that @arielshaqed suggested- both he and I ran into this missing functionality

### Story
Basically, when you run lakectl, it looks at your access key and determines what user you are based on that. That user's authorization policies are applied. It is very common that users make a mistake and specify the wrong access key and it takes a lot of effort to debug.

### Feature Request
It would be nice if we had a lakectl command. Something similar perhaps to https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sts/get-caller-identity.html

### PR DESCRIPTION
**Pull request by ItamarYuran**
Closes #8313

## Change Description

Added functionality to check the user currently configured in lakectl 

### Background

see https://github.com/treeverse/lakeFS/issues/8313
      
### New Feature

The new command prints the user info. The info consists of user id, email (only in case exists) and user creation date

### Testing Details

Added tests esti tests to check both cases of existing and non existing email 

## Additional info
Screenshot 2024-10-30 at 11 57 00

### CODE REVIEW

File: esti/golden/lakectl_identity_with_mail.golden [hunk 1]
```diff
@@ -0,0 +1,3 @@
+User ID:          esti
```
Comment (arielshaqed) [inline esti/golden/lakectl_identity_with_mail.golden:1]: Do we need this file too?

**Review by arielshaqed — APPROVED** [review]
Thanks!Thanks!

File: esti/golden/lakectl_identity_with_mail.golden (hunk 1 above)
Comment (ItamarYuran) [reply esti/golden/lakectl_identity_with_mail.golden:1]: Not really, only in case we solve the esti email issue. We could add it when that happens

**Review by talSofer — APPROVED** [review]
A cool improvement! thank you!

**Comment by N-o-Z** [comment]
@ItamarYuran @arielshaqed  why is this a top level command? Why not put it under auth?

**Pull request merged**